lib/cuda_color_spinor_field.cpp

```diff
@@ -1073,10 +1077,8 @@ namespace quda {
 
           void* ghost_dst = static_cast<char*>(ghost_remote_send_buffer_d[bufferIndex][dim][dir])
             + ghost_precision*ghostOffset[dim][(dir+1)%2];
-          cudaMemcpyAsync(ghost_dst,
-                          my_face_dim_dir_d[bufferIndex][dim][dir],
-                          ghost_face_bytes[dim],
-                          cudaMemcpyDeviceToDevice,
+          qudaMemcpyAsync(ghost_dst, my_face_dim_dir_d[bufferIndex][dim][dir], ghost_face_bytes[dim],
```

Comment:
These device-device copies are the ones I had intentionally not wrapped previously.  IIRC it was because I haven't put in the logic for the peer-to-peer copies using the driver API.  This needs to be checked.